bin/dump-data

```diff
@@ -52,6 +52,15 @@ def find_working_example(schema_name, document_type, rendering_app, publishing_a
   nil
 end
 
+def count_content_items(schema_name, document_type)
+  puts "Counting content items with schema_name: #{schema_name} and document_type: #{document_type}"
+
+  ContentItem.where(
```

Comment:
I think we need to include `rendering_app` and `publishing_app` in the where clause here as I define them to be part of the format in this matrix.